chef/cookbooks/crowbar/templates/default/crowbar-shutdown-services-before-upgrade.sh.erb

```diff
@@ -47,14 +57,15 @@ for type in clone ms primitive; do
     done
 done
 
-<% end %>
-
-# stop cinder-volume at all nodes so the service could be removed from the database later
-if test -e /usr/sbin/rcopenstack-cinder-volume; then
-    log "Stopping cinder-volume service"
-    /usr/sbin/rcopenstack-cinder-volume stop
+ret=$?
```

Comment:
This will only check the result of the last crm call. It's possible an earlier one will have failed.

Reply:
Good point. I'll move the check to crm call.